src/harness/vfs.ts

```diff
@@ -0,0 +1,2064 @@
+/// <reference path="../compiler/commandLineParser.ts"/>
+/// <reference path="./harness.ts" />
+/// <reference path="./core.ts" />
+/// <reference path="./events.ts" />
+/// <reference path="./vpath.ts" />
+/// <reference path="./documents.ts" />
+
+// NOTE: The contents of this file are all exported from the namespace 'vfs'. This is to
+//       support the eventual conversion of harness into a modular system.
+
+namespace vfs {
+    const S_IFMT = 0xf000;

```

Comment:
A comment that these correspond to file `mode` flags (and which flag each one is) would be appreciated.